winforms/src/toga_winforms/widgets/canvas.py

```diff
@@ -4,23 +4,26 @@
 
 from toga.widgets.canvas import Context, FillRule
 from toga_winforms.colors import native_color
-from toga_winforms.libs import (
+from toga_winforms.libs import win_font_family
```

Comment:
As above, this should probably be an import from the `.fonts` direct definition.